doc/user/content/sql/create-source/text-s3.md

```diff
@@ -50,7 +50,17 @@ content:
 
 ### Example TEXT format
 
-First we'll create a source that just ingests all these logs, so that we can execute some
+First we'll create a source that ingests a single file. Because we're only ingesting a
+single file, we only require the `ListObject` [permission](https://materialize.com/docs/sql/create-source/text-s3/#permissions-required):
+
+```sql
+CREATE MATERIALIZED SOURCE single_object
+FROM S3 DISCOVER OBJECTS MATCHING 'logs/2020/12/31/frontend.log' USING BUCKET SCAN 'frontend'
+WITH (region = 'us-east-2', access_key_id='<IAM key>', secret_access_key='<IAM secret>')
```

Comment:
None of the other examples mention the `access_key_id` and `secret_access_key` parameters—I think they ought to be consistent! Best practice (outside of cloud) is to use IAM roles or env vars anyway, rather than hardcoding access keys.

Reply:
makes sense!  The reason I wanted to add the WITH example is that I really  worry that a new user will land on this page, and not be able to interpret the railroad diagram to understand how WITH syntax works 

I can add the params to the examples for parallelism but I worry that it will get wordy.  

i can also drop the WITH altogether, but i suspect we'll lose some users until we can come up with some sort of "CREATE SOURCE for dummies" UI

Reply:
oh, nvm.  i see an example with a WITH.  i'll just drop this